### PR TITLE
Log level

### PR DESCRIPTION
**Pull request by harture**

### CODE REVIEW

File: http/handlers.go [hunk 1]
```diff
@@ -146,19 +145,19 @@ func ErrorHandlerNoLog() func(context.Context, error, http.ResponseWriter) {
...
+func ErrorHandler(logger log.Logger) func(context.Context, error, http.ResponseWriter) {
 	return func(_ context.Context, err error, w http.ResponseWriter) {
 		switch e := errors.Cause(err).(type) {
 		case security.ForbiddenError:
-			logger.Log("ErrorHandler", http.StatusForbidden, "msg", e.Error())
+			logger.Info("ErrorHandler", http.StatusForbidden, "msg", e.Error())
 			w.WriteHeader(http.StatusForbidden)
 		case Error:
-			logger.Log("ErrorHandler", e.Status, "msg", e.Error())
+			logger.Error("ErrorHandler", e.Status, "msg", e.Error())
```
Comment (fperot74) [inline http/handlers.go]: This is an error explicitly created by our components. Shall we log this as Info or Error ?

File: log/log.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package log
+
+import kit_level "github.com/go-kit/kit/log/level"
+import kit_log "github.com/go-kit/kit/log"
+
+type Logger interface {
```
Comment (fperot74) [inline log/log.go]: Add a comment to avoid go-lint warning

File: http/handlers.go (hunk 1 above)
Comment (harture) [reply http/handlers.go]: After discussion, we don't want to log at all at this stage. All errors explicitly created or wrapped by our component would have already been logged. 
We keep logging in the default case only, it should never happen expect for rate limiting and errors we forgot to managed correctly

File: log/log.go (hunk 2 above)
Comment (harture) [reply log/log.go]: fixed

**Pull request merged**